go/base/static/js/test/apps/dialogue/states/choice.test.js

```diff
@@ -212,5 +212,58 @@ describe("go.apps.dialogue.states.choice", function() {
           editMode.$('[data-uuid="choice:new-endpoint"]')));
       });
     });
+
+    describe(".render", function() {
+      it("should display the char count", function() {
+        assert.equal(state.$('.char-count').text().trim(), '96 left of 140');
```

Comment:
I think we should set the model's text, then call `.render()`, then assert that we rendering the right char count. The current test is probably giving us a true positive, but it will be easier for us to verify that we aren't getting a false positive if we explicitly set the model text and explicitly call `.render()`.